README.md

```diff
@@ -295,18 +295,19 @@ If you have received the package and only want to build and run the docker conta
 To run it in a docker container with access to your local network interface, run:
 
 ```bash
-docker run                                                                       \
-    --rm                                                                         \
-    --network     host                                                           \
-    --hostname    signing-service                                                \
-    --name        signing-service                                                \
-    --volume      < path to file >/contract.key:/etc/contract.key:ro             \
-    --restart     on-failure                                                     \
-    concent-signing-service                                                      \
-        --host                      concent.golem.network                        \
-        --port                      9055                                         \
-        --dsn                       https://1234:5678@talkback.golem.network/555 \
-        --ethereum-private-key-path /etc/contract.key
+docker run                                                                               \
+    --rm                                                                                 \
+    --network     host                                                                   \
+    --hostname    signing-service                                                        \
+    --name        signing-service                                                        \
+    --restart     on-failure                                                             \
+    concent-signing-service                                                              \
+        concent.golem.network                              #host                         \
+        85cZzVjahnRpUBwm0zlNnqTdYom1LF1P1WNShLg17cmhN2Us   #concent_public_key in base64 \
```

Comment:
There's a section below that explains the parameters. You should add your remarks there instead of adding comments here.

E.g.:
``` markdown
This asumes that:
(...)
- `85cZzVjahnRpUBwm0zlNnqTdYom1LF1P1WNShLg17cmhN2Us` is Concent's public key encoded in base64.
(...)
```

Reply:
Done.